Makefile

```diff
@@ -163,21 +175,22 @@ install : $(libceed) $(OBJDIR)/ceed.pc
 	$(INSTALL_DATA) $(libceed) "$(DESTDIR)$(libdir)/"
 	$(INSTALL_DATA) $(OBJDIR)/ceed.pc "$(DESTDIR)$(pkgconfigdir)/"
 
-.PHONY: all cln clean print test tst examples astyle install doc
+.PHONY : all cln clean print test tst prove prv examples astyle install doc
+
 cln clean :
 	$(RM) *.o *.d $(libceed)
 	$(RM) -r *.dSYM $(OBJDIR) $(LIBDIR)/pkgconfig
 	$(MAKE) -C examples clean
 	$(MAKE) -C examples/mfem clean
-	cd examples/nek5000; bash make-nek-examples.sh clean; cd ../..;
+	cd examples/nek5000; make-nek-examples.sh clean; cd ../..
 
 distclean : clean
 	rm -rf doc/html
 
 doc :
 	doxygen Doxyfile
 
-astyle :
+style astyle :
```

Comment:
Cosmetic and inessential comment: I'd rather there be one endorsed and documented way to do things than two names that tab completion can display and the user will wonder whether they are really different. I'd rather change the name than have both.